DDR4 OC Guide.md

```diff
@@ -382,54 +398,63 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
+6. Run a memory tester of your choice.
+   * Windows will use ~2000MB so make sure to account for that when entering the amount of RAM to test if the test has manual input. I have 16GB of RAM and usually test 14000MB.
    * Minimum recommended coverage/runtime:
+     * **For AMD run Prime95 Large FFTs and OCCT VRAM with max utilization at the same time to stress the FCLK and to ensure FCLK stability. This should be run after any frequency/FCLK change.**
      * MemTestHelper (HCI MemTest): 200% per thread.
      * Karhu RAMTest: 5000%.
        * In the advanced tab, make sure CPU cache is set to enabled. This will speed up testing by ~20%.
        * Testing for 6400% coverage and a 1 hour duration has an error cover rate of 99,41% and 98,43%, respectively ([Source - FAQ section](https://www.karhusoftware.com/ramtest/)).
      * TM5 anta777 Extreme: 3 cycles.
        * Runtime varies with density. For 16GB RAM, it usually takes between 1.5-2 hours. If you run 32GB RAM you can set the 12th row of the config (Time (%)) to half and you'll get roughly the same runtime as 16GB.
      * OCCT Memory: 30 minutes each for SSE and AVX.
-6. If you crash/freeze/BSOD or get an error, drop the DRAM frequency by a notch and test again.
-7. Save your overclock profile in your UEFI.
-8. From this point on you can either: try to go for a higher frequency or work on tightening the timings.
+     * **You can run more tests such as other TM5 configs to ensure stability. It is recommended to run various tests for maximum error coverage.**
+7. If you crash/freeze/BSOD or get an error, drop the DRAM frequency by a notch and test again.
+9. Save your overclock profile in your UEFI.
+10. From this point on you can either: try to go for a higher frequency or work on tightening the timings.
    * Keep in mind the expectations detailed above. If you're at the limit of your ICs and/or IMC it's best just to tighten the timings.
    
 ## Trying Higher Frequencies
-* This section is applicable if you're not at the limit of your motherboard, ICs and IMC.  
+* This section is applicable if you're not at the limit of your motherboard, ICs, and IMC.  
   This section is not for those who are having trouble stabilising frequencies within the expected range.
-     * Note that some boards have auto rules that can stifle your progress, an example being tCWL = tCL - 1 which can lead to uneven values of tCWL. Reading the [Miscellaneous Tips](#miscellaneous-tips) might give you insight into your particular platform and your motherboards functionality.
+     * Note that some boards have auto rules that can stifle your progress, an example being tCWL = tCL - 1 which can lead to uneven values of tCWL. Reading the [Miscellaneous Tips](#miscellaneous-tips) might give you insight into your particular platform and your motherboard's functionality.
 1. Intel:
-   * Increase VCCSA and VCCIO to 1.25V.
+   * Increase VCCSA and VCCIO to 1.25V. For ADL, VCCIO does not exist.
```

Comment:
If you're going to mention ADL, add all the necessary info about it (gear 1, safe IMC voltages, etc.)

Reply:
Information about those has been added, as well as daily maximums are not fully known due to it being new.